plutus-tx/tutorial/Tutorial.md

```diff
@@ -1,14 +1,18 @@
 # Tutorial
 
-Let's write some code!
```

Comment:
I actually like having some kind of introduction here (even if it's just "Let's write some code!") instead of starting directly with the imports

Reply:
Looks like I merged this too quickly!

Reply:
I'll do a follow-up.